packages/container/lib/container.js

```diff
@@ -23,9 +20,6 @@ function Container(registry, options) {
   this._registry = registry || (function() {
     Ember.deprecate("A container should only be created for an already instantiated registry. For backward compatibility, an isolated registry will be instantiated just for this container.");
 
-    // TODO - See note above about transpiler import workaround.
-    if (!Registry) { Registry = requireModule('container/registry')['default']; }
```

Comment:
No worries!!! I wish it had worked, but I had a few teammates [nombom](https://github.com/rwjblue/dotfiles/blob/master/.zshrc#L52-L53) just after this was published and all hell broke loose in our tests....